src/prob/mld.jl

```diff
@@ -76,6 +82,72 @@ function build_mc_mld(pm::_PM.AbstractPowerModel)
 end
 
 
+"Multinetwork load shedding problem including storage (snap-shot)"
+function build_mn_mc_mld_simple(pm::_PM.AbstractPowerModel)
+    for (n, network) in _PM.nws(pm)
+        variable_mc_branch_power(pm; nw=n)
+        variable_mc_transformer_power(pm; nw=n)
+        variable_mc_gen_power_setpoint(pm; nw=n)
+        variable_mc_bus_voltage(pm; nw=n)
+
+        variable_mc_load_indicator(pm; nw=n, relax=true)
+        variable_mc_shunt_indicator(pm; nw=n, relax=true)
+        variable_mc_storage_power_mi(pm; nw=n, relax=true)
+        _PM.variable_storage_complementary_indicator(pm; nw=n, relax=true)
```

Comment:
This is already added by the call to `variable_mc_storage_power_mi`.

Reply:
Are you sure? I am not seeing it called in the definition of `variable_mc_storage_power_mi`.

Reply:
You are right, with a closer look I think there is a bug in `variable_mc_storage_power_mi`, it should include the complementary indicator.

Here is the template from PowerModels,
```
function variable_storage_power_mi(pm::AbstractPowerModel; kwargs...)
    variable_storage_power_real(pm; kwargs...)
    variable_storage_power_imaginary(pm; kwargs...)
    variable_storage_power_control_imaginary(pm; kwargs...)
    variable_storage_current(pm; kwargs...)
    variable_storage_energy(pm; kwargs...)
    variable_storage_charge(pm; kwargs...)
    variable_storage_discharge(pm; kwargs...)
    variable_storage_complementary_indicator(pm; kwargs...)
end
```
The MC version should be like,
```
function variable_mc_storage_power(pm::AbstractPowerModel; kwargs...)
    variable_mc_storage_power_real(pm; kwargs...)
    variable_mc_storage_power_imaginary(pm; kwargs...)
    variable_mc_storage_current(pm; kwargs...)
    _PM.variable_storage_power_control_imaginary(pm; kwargs...)
    _PM.variable_storage_energy(pm; kwargs...)
    _PM.variable_storage_charge(pm; kwargs...)
    _PM.variable_storage_discharge(pm; kwargs...)
    _PM.variable_storage_complementary_indicator(pm; kwargs...)
end
```

For the on-off version, here is the template from PowerModels,
```
function variable_storage_power_mi_on_off(pm::AbstractPowerModel; kwargs...)
    variable_storage_power_real_on_off(pm; kwargs...)
    variable_storage_power_imaginary_on_off(pm; kwargs...)
    variable_storage_power_control_imaginary_on_off(pm; kwargs...)
    variable_storage_current(pm; kwargs...)
    variable_storage_energy(pm; kwargs...)
    variable_storage_charge(pm; kwargs...)
    variable_storage_discharge(pm; kwargs...)
    variable_storage_complementary_indicator(pm; kwargs...)
end
```
The MC version should be like,
```
function variable_storage_power_mi_on_off(pm::AbstractPowerModel; kwargs...)
    variable_mc_storage_power_real_on_off(pm; kwargs...)
    variable_mc_storage_power_imaginary_on_off(pm; kwargs...)
    variable_mc_storage_current(pm; kwargs...)
    _PM.variable_storage_power_control_imaginary_on_off(pm; kwargs...)
    _PM.variable_storage_energy(pm; kwargs...)
    _PM.variable_storage_charge(pm; kwargs...)
    _PM.variable_storage_discharge(pm; kwargs...)
    _PM.variable_storage_complementary_indicator(pm; kwargs...)
end
```

Reply:
The `mc` version of `variable_storage_power_mi` in the proposal is far off from the current implementation. I will defer to @pseudocubic on this one. For the time being, I've just added the complementarity indicator variables.

Reply:
It might be good to bring the mc function up to date, especially if only that one function needs to be added, it was an oversight on my part

Reply:
Yep, I've added that function. However, there are some other things @ccoffrin suggested, i.e., the addition of `variable_mc_storage_power_real`, `variable_mc_storage_power_imaginary` and removal of `variable_mc_storage_indicator` from `variable_storage_power_mi` that I'm not sure should be changed. In hindsight, it looks like he might have just mistakenly copied much of `variable_mc_storage_power` instead of `variable_storage_power_mi` in his comment, though.